staging/src/k8s.io/apiserver/pkg/apis/apiserver/v1alpha1/types.go

```diff
@@ -376,6 +376,7 @@ type ExtraMapping struct {
 	// subdomain as defined by RFC 1123. All characters trailing the first "/" must
 	// be valid HTTP Path characters as defined by RFC 3986.
 	// key must be lowercase.
+	// Required to be unique.
```

Comment:
this is already validated, right?

Reply:
yes, it's validated

https://github.com/kubernetes/kubernetes/blob/1850794626bb995bb754d54be3328c86ee880ba5/staging/src/k8s.io/apiserver/pkg/apis/apiserver/validation/validation.go#L302-L305